src/components/teams/TeamsLogos.js

```diff
@@ -1,157 +1,212 @@
 export const teamsLogos = [
```

Comment:
this is never used.

I would convert it into an enum and use it throughout the app.
example:

```
export const teamLogos: {
atl: {
  name: "Atlanta Hawks",
    id: "atl",
    numId: 1,
    conferens: "East",
    logoPath: "../../assests/atl.png",
}
}
```

and then you can use: teamLogos[teamId].logoPath to get the logo.

Reply:
The teamsLogos array is used in many components actually. I tried to load the logos PNG files by using the logoPath key in the past, but the PNG isn't loading in that way. So the only part in that array which I didn't use at all is the logoPaths..
I red that in order to load a PNG file to react component I need to import the file first, maybe you know another way to do it?